pulser-simulation/pulser_simulation/hamiltonian.py

```diff
@@ -116,8 +116,12 @@ def basis_check(noise_type: str) -> None:
         local_collapse_ops = []
         if "dephasing" in config.noise_types:
             basis_check("dephasing")
-            coeff = np.sqrt(config.dephasing_rate / 2)
-            local_collapse_ops.append(coeff * qutip.sigmaz())
+            rate = (
+                config.hyperfine_dephasing_rate
+                if self.basis_name == "digital"
```

Comment:
Can you have
```suggestion
                if "digital" in self.basis_name:
```
That would help me in my PR :)

Reply:
Forget about this, there is no need